apps/src/templates/teacherHomepage/ResourceCard.jsx

```diff
@@ -65,11 +62,23 @@ const ResourceCard = React.createClass({
   },
 
   render() {
-    const { title, description, buttonText, link } = this.props;
+    const { title, description, buttonText, link, image } = this.props;
+    const filenameToImgUrl = {
+      "teacher-community": require('../../../static/resource_cards/teachercommunity.png'),
```

Comment:
Yes, I will switch these to absolute paths. I need to merge Course Card images PR first though, so that I have the @cdo/static - no point in doing that work twice.